src/JWT.Extensions.Owin/AppBuilderExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Owin;
+
+namespace JWT
+{
+    public static class AppBuilderExtensions
+    {
+        public static void UseClientCertificateAuthentication(this IAppBuilder appBuilder) =>
```

Comment:
Wrong method name?

Reply:
good spot, thanks, fixed. i'm using this project as a sample: https://github.com/blowdart/idunno.Authentication/tree/master/src/idunno.Authentication.Certificate/